numpy/__init__.py

```diff
@@ -116,26 +116,107 @@
+        double, dtype, e, einsum, einsum_path, empty, empty_like, equal,
+        errstate, euler_gamma, exp, exp2, expm1, fabs, find_common_type, 
+        flatiter, flatnonzero, flexible, 
+        float_, float_power, floating, floor, floor_divide, fmax, fmin, fmod, 
+        format_float_positional, format_float_scientific, format_parser, 
+        frexp, from_dlpack, frombuffer, fromfile, fromfunction, fromiter, 
+        frompyfunc, fromstring, full, full_like, gcd, generic, geomspace, 
+        get_printoptions, getbufsize, geterr, geterrcall, greater, 
+        greater_equal, half, heaviside, hstack, hypot, identity, iinfo, 
+        indices, inexact, inf, infty, inner, int_,
+        intc, integer, invert, is_busday, isclose, isfinite, isfortran,
+        isinf, isnan, isnat, isscalar, issctype, issubdtype, lcm, ldexp,
+        left_shift, less, less_equal, lexsort, linspace, little_endian, log, 
+        log10, log1p, log2, logaddexp, logaddexp2, logical_and, logical_not, 
+        logical_or, logical_xor, logspace, longcomplex, longdouble, 
+        longfloat, longlong, matmul, max, maximum, maximum_sctype, 
+        may_share_memory, mean, min, min_scalar_type, minimum, mod, 
+        modf, moveaxis, multiply, nan, nbytes, ndarray, ndim, nditer, 
+        negative, nested_iters, newaxis, nextafter, nonzero, not_equal,
+        number, obj2sctype, object_, ones, ones_like, outer, partition,
+        pi, positive, power, printoptions, prod, product, promote_types, 
+        ptp, put, putmask, rad2deg, radians, ravel, rec, recarray, reciprocal,
+        record, remainder, repeat, require, reshape, resize, result_type, 
+        right_shift, rint, roll, rollaxis, round, round_, sctype2char, 
+        sctypeDict, sctypes, searchsorted, set_printoptions,
+        set_string_function, setbufsize, seterr, seterrcall, shape,
+        shares_memory, short, sign, signbit, signedinteger, sin, single, 
+        singlecomplex, sinh, size, sometrue, sort, spacing, sqrt, square, 
+        squeeze, stack, std, str_, string_, subtract, sum, swapaxes, take,
+        tan, tanh, tensordot, timedelta64, trace, transpose, 
+        true_divide, trunc, typecodes, ubyte, ufunc, uint, uintc, ulonglong, 
+        unicode_, unsignedinteger, ushort, var, vdot, void, vstack, where, 
+        zeros, zeros_like, _get_promotion_state, _set_promotion_state
+    )
+
+    sized_aliases = ["int8", "int16", "int32", "int64", "intp", 
+                     "uint8", "uint16", "uint32", "uint64", "uintp",
+                     "float16", "float32", "float64", "float96", "float128",
+                     "complex64", "complex128", "complex192", "complex256"]
+    
+    for sa in sized_aliases:
+        try:
+            globals()[sa] = getattr(core, sa)
+        except AttributeError:
+            pass
+    del sa, sized_aliases
+
     from . import lib
     # NOTE: to be revisited following future namespace cleanup.
     # See gh-14454 and gh-15672 for discussion.
```

Comment:
I believe this PR is the namespace cleanup this note references, so best to delete it.

Reply:
Sure! Removed